BartIngress/Program.cs

```diff
@@ -52,8 +52,8 @@ public static void LoadConfiguration()
             OmfServices.SendOmfType(typeof(BartStationEtd));
 
             // Send OMF Container Message
-            var data = BartApi.GetRealTimeEstimates(Settings.BartApiKey, Settings.BartApiOrig, Settings.BartApiDest);
-            var typeId = ClrToOmfTypeConverter.Convert(typeof(BartStationEtd)).Id;
+            System.Collections.Generic.Dictionary<string, System.Collections.Generic.IEnumerable<BartStationEtd>> data = BartApi.GetRealTimeEstimates(Settings.BartApiKey, Settings.BartApiOrig, Settings.BartApiDest);
```

Comment:
using here as well, I'm guessing VS used the full spec automatically

Reply:
good catch, added the using statement